packages/shared/src/types/targeting/header.ts

```diff
@@ -1,18 +1,30 @@
-import { PageTracking } from './shared';
+import { PageTrackingSchema } from './shared';
+import { z } from 'zod';
```

Comment:
Unfortunately we cannot add zod to files in /targeting because they get used by the dotcom package.
I need to find a way to prevent this as it's not at all obvious!

Reply:
Ahh yeah I thought this might be the case - is there some way to confirm this is happening? I tried building on the main branch and then on this one and couldn't see a difference in bundle size

Reply:
I think the solution here might be to add another package under `/packages` e.g `validation` that does all of the `zod` stuff which will keep it clear of anything the `dotcom` package uses

Reply:
That's a good suggestion!
So far I've just looked in the generated js in /packages/dotcom/dist/ to make sure it's not bringing in unwanted code, e.g. zod. But this isn't really acceptable!